core/src/context.rs

```diff
@@ -58,18 +59,17 @@ pub struct Issue {
 
 impl fmt::Display for Issue {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-        if let Some(input) = &self.input {
-            write!(
-                f,
-                "{} -> {}: {}, input: {}",
-                self.source, self.target, self.description, input
-            )
-        } else {
-            write!(
+        match self.input.as_ref().map(String::as_str) {
+            Some("input") | None => write!(
```

Comment:
What does this matching against `Some("input")` do? When is the `input` argument equals to `input`? I don't think you did something wrong, I'm just curious

Reply:
Oops! I think it actually is a type and should say something along the lines of "<unknown>" and not "input". Good catch! I'll fix that before I rebase